### ISSUE TITLE
Finding Minimum Element in a Stack in O(1) time

### ISSUE THREAD

**Issue by sumitgsh** [HarshCasper/NeoAlgo#2972]
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)
Yes of course!!
## Motivation

(Please outline the motivation for the proposal.)
Design a stack that, in addition to Push and Pop, should have a function min that returns the minimum element in O(1) time

## Pitch

(Please explain why this feature should be implemented and how it would be used.)
Basically returning the min element in the stack is an Easy Task but when time is the barrier the problem becomes interesting.
This type of problem will make everyone realize the variety of ways DS can be used.

**Comment by sumitgsh** [comment]
@siddharth25pandey @HarshCasper @iamrajiv can u please assign me the issue !!
Would love to work on C /Java as a part of GsSoc21
